.mage/js.go

```diff
@@ -81,29 +81,27 @@ func (js Js) node() (func(args ...string) error, error) {
 	}, nil
 }
 
-func (js Js) execFromNodeBin(cmd string, verbose bool) (func(args ...string) error, error) {
+func execFromNodeBin(cmd string, verbose bool, appendArgs ...string) (func(args ...string) error, error) {
```

Comment:
Please revert change in this function and use `withArguments` from https://github.com/TheThingsNetwork/lorawan-stack/pull/2751